<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeSwellDialog</class>
 <widget class="QDialog" name="VolumeSwellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Swell</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="startVolumeLabel">
       <property name="text">
        <string>Start Volume:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="startVolumeLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QToolButton" name="volumeOffButtonStart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Off</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pppButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_ppp.png</normaloff>:/images/dynamic_ppp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ppButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_pp.png</normaloff>:/images/dynamic_pp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_p.png</normaloff>:/images/dynamic_p.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mpButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_mp.png</normaloff>:/images/dynamic_mp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mfButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_mf.png</normaloff>:/images/dynamic_mf.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_f.png</normaloff>:/images/dynamic_f.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ffButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_ff.png</normaloff>:/images/dynamic_ff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fffButtonStart">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_fff.png</normaloff>:/images/dynamic_fff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="endVolumeLabel">
       <property name="text">
        <string>End Volume:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="endVolumeLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QToolButton" name="volumeOffButtonEnd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Off</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pppButtonEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_ppp.png</normaloff>:/images/dynamic_ppp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ppButtonEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_pp.png</normaloff>:/images/dynamic_pp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pButtonEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_p.png</normaloff>:/images/dynamic_p.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mpButtonEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_mp.png</normaloff>:/images/dynamic_mp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="mfButtonEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_mf.png</normaloff>:/images/dynamic_mf.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fButtonEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_f.png</normaloff>:/images/dynamic_f.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ffButtonEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_ff.png</normaloff>:/images/dynamic_ff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fffButtonEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../build/resources.qrc">
           <normaloff>:/images/dynamic_fff.png</normaloff>:/images/dynamic_fff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="swellDurationLabel">
       <property name="text">
        <string>Swell Duration:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="overCurNoteOpt">
         <property name="text">
          <string>Current Note</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="overFollowingNotesOpt">
         <property name="text">
          <string>Current Note and Following Notes</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="numberOfNotesLabel">
       <property name="text">
        <string>Number of Notes:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="numNotesSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../build/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VolumeSwellDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VolumeSwellDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
